pkgs/by-name/fi/fio-plot/package.nix

```diff
@@ -0,0 +1,6 @@
+{
+  python3Packages,
+}:
+(python3Packages.toPythonApplication python3Packages.fio_plot).overrideAttrs (old: {
+  meta.mainProgram = "fio-plot";
```

Comment:
You can check the output with `nix eval -f ./. fio-plot.meta`
```suggestion
  meta.mainProgram = old.meta // { mainProgram = "fio-plot"; };
```

Reply:
Ah, nice. Changed!